src/Workspaces/Core/Portable/Storage/SQLite/v2/SQLitePersistentStorage.Accessor.cs

```diff
@@ -64,10 +64,9 @@ public Accessor(SQLitePersistentStorage storage)
 
             [PerformanceSensitive("https://github.com/dotnet/roslyn/issues/36114", AllowCaptures = false)]
```

Comment:
would be nifty if something validated this automatically.  for now though, we have the attribute declaring intent, and we can ensure it is followed by use of `static` lambdas.

Reply:
That's the plan for [PerfSensitiveAnalyzer](https://github.com/dotnet/roslyn-analyzers/tree/master/src/PerformanceSensitiveAnalyzers), we just never get time to work on that